node/test/graphql/query/me/training-data-all.spec.ts

```diff
@@ -107,7 +107,7 @@ describe('training data all', () => {
     expect(response.body.data.me.allTrainingData.isTrainable).to.eql(true);
   });
 
-  it(`returns all training data for lesson`, async () => {
+  it(`returns all non-invalidated training data for lessons`, async () => {
```

Comment:
let's keep the original test as it was but then add a test like

```typescript
it(`excludes invalidated responses from training data`, async () => {
  // get the training data
  // expect includes some specific example
  // call invalidate on the example above
  // get the training data again
  // expect DOES NOT include the invalidated example

})

Reply:
wouldn't they just do and return the exact same thing though? is there a point to having two of the same test?